### DISCUSSION TITLE
How TGC dev catch auto CR ...!!!!!!!

### DISCUSSION THREAD

**Discussion by Evil-0-King** [thatskymod/Sky-CotL-Scripts#97] [General]
"These are OBB (hidden) candles, if any one get wax from it, that means he is hacking...."

![image](https://github.com/thatskymod/Sky-CotL-Scripts/assets/97114973/d5a80ecb-a37c-4c62-bf78-86a4de1319d0)

**Comment by thatskymod** [comment]
OOBs or '[Out-of-Bounds](https://sky-children-of-the-light.fandom.com/wiki/Out_of_Bounds)' experiences (not OBBs) are not against TGC's [Terms of Service](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/460-eula-terms-of-service/), similar to how it's not forbidden to join a friend who is in those areas by accident. In fact, OOBs are a unique characteristic of the game, which is why they remain unpatched for so long.

However, visiting unreleased realms not intended for the public (such as exploring a seasonal area early or accessing the Halloween area outside of Halloween) is considered a violation.

Look up 'OOB ToS' or 'OOB illegal' on [Discord](https://discord.com/invite/thatskygame) for more information.
